photon-core/src/main/java/org/photonvision/vision/camera/TestSource.java

```diff
@@ -34,9 +34,8 @@ public class TestSource extends VisionSource {
     public TestSource(CameraConfiguration config) {
         super(config);
 
-        if (getCameraConfiguration().cameraQuirks == null)
-            getCameraConfiguration().cameraQuirks =
-                    QuirkyCamera.getQuirkyCamera(config.usbVID, config.usbVID, config.baseName);
+        getCameraConfiguration().cameraQuirks =
```

Comment:
does this blow away arducam quirks?

Reply:
It does, missed undoing that change, will fix.